### PR TITLE
Migrate debian-oval and ubuntu importer

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

**Review by pombredanne — CHANGES_REQUESTED** [review]
Looking good... do you mind to rebase on the latest code?

**Comment by pombredanne** [comment]
@TG1999 let's rebase one more time before merging!

File: vulnerabilities/importers/debian_oval.py [hunk 1]
```diff
@@ -8,38 +8,29 @@
...
 
 import requests
 
 from vulnerabilities.importer import OvalImporter
-from vulnerabilities.package_managers import DebianVersionAPI
-from vulnerabilities.utils import create_etag
 
 
 class DebianOvalImporter(OvalImporter):
+    spdx_license_expression = "LicenseRef-scancode-unknown"
```
Comment (pombredanne) [inline vulnerabilities/importers/debian_oval.py:19]: This should be:
```suggestion
    spdx_license_expression = "LicenseRef-scancode-other-permissive"
```
and we should have the exact the same value for https://github.com/nexB/vulnerablecode/blob/ae609a857c747c45c8391b9b0fd3c54b0b30d921/vulnerabilities/importers/debian.py#L42 ...
The license URL and MIT does not apply here IMHO and we should not have a license URL

And the notice in debian should also exist here: https://github.com/nexB/vulnerablecode/blob/ae609a857c747c45c8391b9b0fd3c54b0b30d921/vulnerabilities/importers/debian.py#L44

File: vulnerabilities/importers/debian_oval.py [hunk 2]
```diff
@@ -8,38 +8,29 @@
...
         super().__init__(*args, **kwargs)
         # we could avoid setting translations, and have it
         # set by default in the OvalParser, but we don't yet know
         # whether all OVAL providers use the same format
         self.translations = {"less than": "<"}
-        self.pkg_manager_api = DebianVersionAPI()
 
     def _fetch(self):
-        releases = self.config.releases
+        releases = ["wheezy", "stretch", "jessie", "buster", "bullseye"]
```
Comment (pombredanne) [inline vulnerabilities/importers/debian_oval.py:29]: Please enter an issue to find a way to get alerted when there are new and retied Debian releases

File: vulnerabilities/utils.py [hunk 3]
```diff
@@ -318,7 +318,7 @@ def get_reference_id(url: str):
 def resolve_version_range(
     affected_version_range: VersionRange,
     package_versions: List[str],
-    ignorable_versions: List[str],
+    ignorable_versions: List[str] = [],
```
Comment (pombredanne) [inline vulnerabilities/utils.py:321]: This is a global... make it a tuple to avoid issues.

**Review by pombredanne — APPROVED** [review]
LGTM except for a small nit... Please merge!

File: vulnerabilities/importers/debian_oval.py (hunk 2 above)
Comment (TG1999) [reply vulnerabilities/importers/debian_oval.py:29]: https://github.com/nexB/vulnerablecode/issues/967

**Pull request merged**